README.md

```diff
@@ -1,43 +1,191 @@
 # Pingh
 
-An app notifying of new GitHub `@mention`s.
+A [GitHub App](https://github.com/apps/pingh-tracker-of-github-mentions) that notifies users 
+of new GitHub `@mention`s. It also runs as a system tray application on macOS.
 
-## Status
+- Displays a list of recent `@mention`s for a logged-in GitHub user.
+- Marks mentions as “read” or snooze them for later.
+- Sends notifications for new mentions and for mentions whose snooze time has expired.
+- Enters DND mode to turn off notifications.
 
-Currently, this is a work-in-progress example.
+## Tech
 
-## Planned Features
+Server stack:
 
-- [ ] Display the list of recent `@mention`s to a logged in GitHub user via a Chrome extension.
-- [ ] Mark the mentions as "read", or snooze them until later.
-- [ ] Enter DND mode.
+- Spine Event Engine 1.9.0.
+- Ktor 2.3.11. 
+- Kotlin 1.9.20.
+- JDK 11.
+- Gradle 6.9.4.
+- Google Cloud.
+- Docker.
 
-## Tech
+Client stack:
+
+- Compose Multiplatform 1.6.11.
+- Kotlin 1.9.20.
+- JDK 17.
+- Gradle 8.8.
+
+## Project structure
+
+The project consists of the following modules:
+
+- `github` provides value objects to represent the GitHub context, 
+  along with descriptions of the JSON responses returned by the GitHub REST API.
+- `sessions` provides an implementation of the Sessions bounded context, 
+  which includes user session management and authentication via GitHub.
+- `mentions` provides an implementation of the Mentions bounded context,
+  which includes managing mention statuses and 
+  the process of retrieving new user mentions from GitHub.
+- `clock` emulates an external system by sending the current time to the server.
+- `server` configures the server environment, sets up the server, and starts it.
+  This module also enables interaction with [Google Cloud](#google-cloud-deployment).
+- `client` provides process states and flows for the client application.
+- `desktop` provides the user interface created with Compose Multiplatform.
+
+There are several auxiliary modules available for testing:
 
-* JDK 11 for server and JDK 17 for client.
-* Spine Event Engine.
-* Kotlin.
-* Google Cloud.
+- `testutil-sessions` allows authentication to the app without using the GitHub REST API.
+- `testutil-mentions` allows retrieving new user mentions without using the GitHub REST API.
 
-## Run locally
+For a detailed analysis of the processes within domain contexts, 
+refer to the [#EventStorming documentation](./EventStorming.md).
+
+## Local run
+
+The application server can be run locally, and the client application distribution 
+can also be built locally.
+
+When running the application locally, consider the following:
+
+- In-memory storage is used, so data will be lost between server restarts.
+- The GitHub App ID and secret must be manually specified, meaning secrets are stored directly 
```

Comment:
Both GitHub App ID and GitHub ... secret must be specified explicitly by editing `...` file.